app/views/works/form.html.erb

```diff
@@ -36,8 +36,8 @@
     <% end %>
 
     <% component.with_pane(tab_name: :related_content, label: t('works.edit.panes.related_content.label'), form:) do %>
-      <%= render Elements::Forms::NestedComponent.new(form:, model_class: RelatedWorkForm, field_name: :related_works, form_component: RelatedWorks::EditComponent) %>
-      <%= render Elements::Forms::NestedComponent.new(form:, model_class: RelatedLinkForm, field_name: :related_links, form_component: RelatedLinks::EditComponent) %>
+      <%= render Elements::Forms::NestedComponent.new(form:, model_class: RelatedWorkForm, field_name: :related_works, form_component: RelatedWorks::EditComponent, classes: 'mb-3 p-3 row border border-3 border-light-subtle border-opacity-75') %>
```

Comment:
Feel like these classes should be abstracted behind a boolean argument like `bordered` on EditComponent. Otherwise, we will be repeating the same CSS all over the place.

Reply:
I can do that, I'll work on it this evening.